### ISSUE TITLE
Specify and/or upgrade KaTeX version

### ISSUE THREAD

**Issue by rubenvannieuwpoort** [graemephi/goldmark-qjs-katex#8]
First, let me start off by stating how much I like both goldmark and this extension -- I am making a static site generator, and it's a blast thanks to this!

I ran into a small issue where KaTeX formulas seem to be rendered with the wrong font after migrating my blog. Looking at the diff between the previous version, I see that `goldmark-qjs-katex` renders the math with `<span class="mord mathnormal">` whereas it would use `<span class="mord mathdefault">` previously. A [quick search](https://github.com/KaTeX/KaTeX/issues/2347) suggests that this style has changed in some version of KaTeX.

This makes sense as I just included the same KaTeX stylesheet as I used before, without paying attention to the version. I would think that it would work if I include the right version of the KaTeX stylesheet. However, I'm not sure where I can find the version that is used in `goldmark-qjs-katex`. It might be good to specify it in the README so that people can include the right version.

It might also be desirable to upgrade the KaTeX version used. I'll give this a try.

**Comment by graemephi** [comment]
I have two thoughts on this. One is I'm happy to merge the PR if you get the tests working. The tests are mostly to make sure goldmark handles switching between markdown and latex in a nice way, and making any modifications to the goldmark extension itself is where that might break: that's the case those tests are supposed to be helping you with. Updating KaTeX, it's just extra busywork. So even though `gen_test.go` is a generated thing it says not to edit, it might be easiest to just modify the test cases in there to use the new class name. If that works, that's fine. 

In the worse case here it might turn out the test generator is relying on KaTeX behaviour that has changed in the new version and it may fail tests that should pass (the `normalize` function in the tests is meant to handle that kind of thing). But again my feeling is if that's the case, the easiest course of action will be a big find and replace to update the generated test cases

Second thing is I made this so I could throw it into Hugo and they've recently added the same functionality themselves using wasm instead of qjs. I don't know if this is more or less hassle than what you're trying here but what they're doing might be worth a look at

**Comment by rubenvannieuwpoort** [comment]
Thanks for your reply! 

Regarding looking at Hugo: I have tried to use it before but didn't like it too much at the time. Since I have invested a bit in goldmark and the qjs-katex extension now, and they seem to work great for my purposes I don't intend to look at other options now.

I worked on https://github.com/graemephi/goldmark-qjs-katex/pull/9 a bit. It took a bit longer than expected but the tests pass now and I don't plan to make more changes to the PR unless you have comments.

I had to manually replace `pandoc`s output for some cases. After manually replacing those testcases I don't see a change in the PR diff for them, so I figure these tests were only passing before because they were manually edited in `gen_test.go` as well. (Maybe the `DO NOT EDIT` message should go, or the failing tests should be moved to a different test file which is not autogenerated?).
